test/mix/tasks/ecto.gen.migration_test.exs

```diff
@@ -68,4 +68,27 @@ defmodule Mix.Tasks.Ecto.Gen.MigrationTest do
   test "raises when missing file" do
     assert_raise Mix.Error, fn -> run ["-r", to_string(Repo)] end
   end
+
+  test "generates migration within umbrella app without raising" do
```

Comment:
The test is good but I am afraid it is too brittle. So I think we should rather remove it... sorry! :D